data/json/monsters/insect_spider.json

```diff
@@ -1356,5 +1356,34 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_mantis",
+    "type": "MONSTER",
+    "name": { "str": "giant preying mantis" },
+    "description": "A preying mantis, the size of a horse. If you weren't looking closely, you might mistake it for a dead tree.",
+    "default_faction": "mantis",
+    "bodytype": "flying insect",
+    "species": [ "INSECT" ],
+    "volume": "550 L",
+    "weight": "275 kg",
```

Comment:
This is half as dense as water, which seems insufficiently dense for a living creature.  I'd expect a density around .75-1.1 kg/m^3, so it should weigh more or have less volume.

Reply:
According to [this](https://biology.stackexchange.com/questions/59528/what-is-the-body-density-of-insects-on-average), insects typically have density lower than 0.3g/ml, which indeed is far less dense than water. Whether that data can be extrapolated for an imaginary insect the size of a horse is another matter, but I doubt there will be any real life data for that.

Reply:
According to [this](https://biology.stackexchange.com/questions/59528/what-is-the-body-density-of-insects-on-average), insects typically have density lower than 0.3g/mL, which is indeed far less dense than water. Whether that data can be extrapolated to an imaginary insect the size of a horse is another matter, but I doubt there will be any real life data for that...

Reply:
The mutant variant are only insect shaped, internally they have a bunch of stuff vertebrate-like and should probably be denser than regular insects.